rpmlint/checks/FileDigestCheck.py

```diff
@@ -20,6 +20,17 @@ def __init__(self, config, output):
             if dg_type not in self.digest_configurations:
                 raise KeyError(f'FileDigestGroup type "{dg_type}" is not '
                                f'supported, known values: {list(self.digest_configurations)}')
+            # expand skip digests
+            for skip_digest in digest_group.get('skips', []):
```

Comment:
Didn't we want to use a more intuitive name than `skips` here? Like `nodigests` or something alike.

Reply:
Ah only now I see that you renamed it it a follow-up commit. Can you sqash these two commits so we get a cleaner history?